### DISCUSSION TITLE
lost 100 plots Moving HDD with plots from Linux system to a Windows system.

### DISCUSSION THREAD

**Discussion by jmakme** [Chia-Network/chia-blockchain#5659] [Support]
Long story short, after 3 days of running fine, my Ubuntu machine started getting system errors and shutting down randomly. Linux errors, not chia. I lost a lot of plots over a few days this way, but managed to store 112 of them across 2 different 12 tb internal HDD's.
Being fairly new to Linux, i decided to cut my losses and just install windows and restart the whole thing in a more familiar environment. 
Installed windows fine on a 250g boot drive. 
Once into windows, the HDD's showed no partition, no files. 12tb free.
At this stage I panicked and plugged them back into a different machine that was still running Ubuntu. Figuring, theres no way a Linux machine cant read a Linux partition and files. 
Alas, Same thing. No partitions, no data.
(SIDE NOTE: The plots were stupidly not put in a folder on the destination drive. simply into the /E  or /D directories)

Im very lost on where to go from here. I am inclined to start up TestDisk on the Ubuntu machine and going that route, but as there is no information on recovering .plot files, I am unsure if this is even a viable option.

Kinda freaking out over losing the 112 plots on top of the 3 days of losses due to crashes.

Any and all help is appreciated.

**Comment by BrandtH22** [comment]
@jmakme , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
